packages/grafana-schema/src/schema/dashboard/v0alpha1/dashboard_object_gen.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * This file was generated by grafana-app-sdk. DO NOT EDIT.
+ */
+import { Spec } from './types.spec.gen';
+import { Status } from './types.status.gen';
+
+export interface Metadata {
+    name: string;
+    namespace: string;
+    generateName?: string;
+    selfLink?: string;
+    uid?: string;
+    resourceVersion?: string;
+    generation?: number;
+    creationTimestamp?: string;
+    deletionTimestamp?: string;
+    deletionGracePeriodSeconds?: number;
+    labels?: Record<string, string>;
+    annotations?: Record<string, string>;
```

Comment:
For those we currently use this type; https://github.com/grafana/grafana/blob/main/public/app/features/apiserver/types.ts#L62 which is independent yet aligned with the annotations specified in the dashboards api implementation. 

Happy to continue using that type, but think would be great if we could gen these too to keep the types aligned.

Reply:
Yeah, same as above, we need to unify metadata implementations.